app/AppComponents.scala

```diff
@@ -24,12 +24,10 @@ class AppComponents(context: Context)
 
   val config = new Config(context.initialConfiguration)
 
-  val panDomainRefresher = new PanDomainAuthSettingsRefresher(
+  val panDomainRefresher = PanDomainAuthSettingsRefresher(
     domain = config.domain,
     system = config.pandaSystem,
-    bucketName = config.pandaBucketName,
-    settingsFileKey = config.pandaSettingsFile,
-    s3Client = AWS.S3Client
+    S3BucketLoader.forAwsSdkV1(AWS.S3Client, "pan-domain-auth-settings")
```

Comment:
I think I made a mistake here: the second parameter for `S3BucketLoader.forAwsSdkV1` is called `bucketName`, so presumably this should’ve been `config.pandaBucketName` rather than this literal string I copied from another example?

Reply:
Ah yes, that probably should have been `config.pandaSettingsFile` for the smallest possible change, tho' `"pan-domain-auth-settings"` is almost certainly the value `config.pandaSettingsFile` would give!

Reply:
I’ll prepare a new PR that uses the config value or removes it, and removes the settingsFileKey config value.